vdom/src/com/vdom/core/IndirectPlayer.java

```diff
@@ -212,7 +212,13 @@ public int selectInt(MoveContext context, Card responsible, int maxInt, Object e
                 options[i + 1] = int_options[i];
             }
         }
-        return int_options[selectOption(context, responsible, options)];
+        int choice = selectOption(context, responsible, options);
```

Comment:
Another option here would be to have the remote player return 0 instead of -1 when selecting options.